src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Key.java

```diff
@@ -108,6 +108,8 @@ abstract class P11Key implements Key, Length {
 
     private static final boolean DISABLE_NATIVE_KEYS_EXTRACTION;
 
+    private static boolean isPlainPBEKeySupportInFIPS = false;
```

Comment:
A static field is not appropriate for this purpose; `key != null` is a suitable replacement.

Reply:
Updated